src/Sentry.Extensions.AI/SentryAIConstants.cs

```diff
@@ -0,0 +1,67 @@
+namespace Sentry.Extensions.AI;
+
+internal static class SentryAIConstants
+{
+    /// <summary>
```

Comment:
Maybe worth adding a reference to the source code where we found these, in case they change in the future